lambda-loader/src/api.py

```diff
@@ -16,7 +16,7 @@ def getLeaderboardSnapshot():
         accounts = json.loads(r.text)['leaderboard']['rows']
 
         for account in accounts:
-            name = account['accountid'].encode('utf-8').lower()
+            name = account['accountid'].encode('utf-8').lower() # Why is this converted to bytes?
```

Comment:
I believe I needed it to handle some of the asian language names but maybe it's not actually needed.

Reply:
Yeah I am curious if there are some other bytes operations down the road on request workflows.

DynamoDB can of course accept strings with any utf-8 characters, so strange characters can be part of primary keys.

If we need for it to be bytes, we need to recreate the database and change the type of the primary key from string to bytes.